frontend/src/app/containers/HomePage/index.scss

```diff
@@ -0,0 +1,27 @@
+@import '../../../styles/_utils';
+
+.landingExperiments.notvisible {
+  h1,
+  h2 {
+    cursor: pointer;
+    width: 100%;
+    margin: 0;
+    padding: $grid-unit * 2 0;
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    background: linear-gradient($transparent-black-2 40px, $transparent-black-15);
```

Comment:
This background being transparent doesn't really work; it can clash with whatever happens to be in the page.
![screen shot 2017-12-18 at 11 26 32 am](https://user-images.githubusercontent.com/46048/34116357-60576750-e3e6-11e7-9489-b144b947a5c6.png)

Reply:
Yeah, like I said in a previous comment, I didn't put much work into the styles at all because I don't have a UX spec for it. I don't think the styles I have are good at all